AsyncDisplayKitTests/ASImageNodeSnapshotTests.m

```diff
@@ -46,12 +46,14 @@ - (void)testForcedScaling
   // Snapshot testing requires that node is formally laid out.
   imageNode.style.width = ASDimensionMake(forcedImageSize.width);
   imageNode.style.height = ASDimensionMake(forcedImageSize.height);
-  [imageNode layoutThatFits:ASSizeRangeMake(CGSizeZero, forcedImageSize)];
+  ASDisplayNodeSizeToFitSize(imageNode, forcedImageSize);
+  [imageNode layoutIfNeeded];
   ASSnapshotVerifyNode(imageNode, @"first");
```

Comment:
Doesn't SnapshotVerify force a layoutIfNeeded pass?  IIRC it does because this is required to properly set up and wait on display to finish...

If so, we should remove the layoutIfNeeded calls for these.